ui/StatusQ/src/StatusQ/Components/StatusItemSelector.qml

```diff
@@ -93,7 +65,7 @@ Rectangle {
         }
        \endqml
     */
-    property var itemsModel: ListModel { }
+    property var itemsModel
```

Comment:
Just `model`?

Reply:
yeah, it was `itemsModel` so far but just `model` is better indeed, refactored